cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -296,6 +296,24 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
```

Comment:
need same changes in common.yaml and e2e manifests also

Reply:
was already on it, common is already addressed. Looking into e2e.

Reply:
done. thanks .